src/utils/interfaces.ts

```
@@ -1372,7 +1372,8 @@ export interface InterfaceUserInfoPG {
 /**
  * Extends InterfaceBaseEvent with additional properties for an organization event list item.
  */
-export interface InterfaceQueryOrganizationEventListItem extends InterfaceBaseEvent {
+export interface InterfaceQueryOrganizationEventListItem
+  extends InterfaceBaseEvent {
```

ESLint

[error] Delete `⏎·`

(prettier/prettier)